desktop/core/src/desktop/js/reactComponents/MetricsComponent/MetricsComponent.test.tsx

```diff
@@ -96,4 +96,15 @@ describe('MetricsComponent', () => {
       });
     }
   });
+
+  // Test for ensuring metrics starting with 'python.gc', 'multiprocessing', and 'auth' are not displayed
+  test('ensuring metrics starting with auth, multiprocessing and python.gc are not displayed', async () => {
+    render(<MetricsComponent />);
+
```

Comment:
I take it the data comes from a setup function, perhaps add a comment and an assertion that some data we'd expect to see is also in the rendered component.

(This test will succeed when there's no data at all)

Reply:
Thanks for the review Johan.
I have now added the python.gc etc in the mock api.